lib/util.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * @fileoverview Package-private helpers for the installer.
+ */
+
+'use strict'
+
+var cp = require('child_process')
+var fs = require('fs-extra')
+var helper = require('./slimerjs-core')
+var kew = require('kew')
+var path = require('path')
+
+var DEFAULT_CDN = 'https://github.com/graingert/slimer-downloads/releases/download/'
```

Comment:
Yeah, I removed the download functionality, let me clean up these unused vars. Thanks!